airflow/dags/sync_elavon/elavon_to_gcs_raw.py

```diff
@@ -0,0 +1,62 @@
+# ---
+# python_callable: mirror_raw_files_from_elavon
+# provide_context: true
+# ---
+import os
+
+import paramiko
+import pendulum
+from calitp_data_infra.auth import get_secret_by_name
+from calitp_data_infra.storage import get_fs
+
+CALITP__ELAVON_SFTP_HOSTNAME = os.environ["CALITP__ELAVON_SFTP_HOSTNAME"]
+CALITP__ELAVON_SFTP_PORT = os.environ["CALITP__ELAVON_SFTP_PORT"]
+CALITP__ELAVON_SFTP_USERNAME = os.environ["CALITP__ELAVON_SFTP_USERNAME"]
+CALITP__ELAVON_SFTP_PASSWORD = get_secret_by_name("CALITP__ELAVON_SFTP_PASSWORD")
+
+
+def mirror_raw_files_from_elavon():
+    """
+    Download Elavon transaction records from SFTP and write raw files to GCS for
+    further processing in another job
+    """
+
+    # Establish connection to SFTP server
+    client = paramiko.SSHClient()
+    client.set_missing_host_key_policy(paramiko.AutoAddPolicy())
+    client.connect(
+        hostname=CALITP__ELAVON_SFTP_HOSTNAME,
+        port=CALITP__ELAVON_SFTP_PORT,
+        username=CALITP__ELAVON_SFTP_USERNAME,
+        password=CALITP__ELAVON_SFTP_PASSWORD,
+    )
+
+    # Create SFTP client and navigate to data directory
+    sftp_client = client.open_sftp()
+    sftp_client.chdir("/data")
+
+    # Initialize GCS connection
+    fs = get_fs()
+
+    # Initiailize extract time used by all files
+    ts = str(pendulum.now())
```

Comment:
can you use `dt.to_iso8601_string()` like `PARTITION_SERIALIZERS`?

Reply:
Resolved in [1be64ea](https://github.com/cal-itp/data-infra/pull/2376/commits/1be64eabaf9980b0b2904124b125ed395f66912e)